debian/jessie/Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM debian:jessie
+
+LABEL maintainer me@danvaida.com
+
+RUN DEBIAN_FRONTEND=noninteractive \
+    apt-get update -y && \
+    apt-get install --fix-missing && \
+    apt-get install -y gcc='4:4.9.2-2' \
+                       libffi-dev='3.1-2+b2' \
+                       libssl-dev='1.0.1t-1+deb8u6' \
+                       python=2.7.9-1 \
+                       python-dev=2.7.9-1 \
+                       python-pip=1.5.6-5 \
+                       python-yaml=3.11-2
+
+RUN apt-get remove -y --purge python-cffi
```

Comment:
indeed, every command creates an (in this case, useless) intermediary layer. thanks for reminding me! this led to reducing the size with >50%!
```
$ docker images | grep jessie
jessie_proper_clean              latest              57dcb031d777        7 seconds ago       200 MB
jessie_fake_clean                latest              393ab4ce6832       30 minutes ago      447 MB
```